### ISSUE TITLE
"An unexpected error has occured and has been added to the log file." + amd&nvidia gpu fan control

### ISSUE THREAD

**Issue by Tron-Creeper2** [Rem0o/FanControl.Releases#1215]
**Describe the bug**
[A clear and concise description of what the bug is.  Screenshots are worth a thousand words. Hardware support requests do NOT count as a bug. See https://github.com/Rem0o/FanControl.Releases#issues-and-hardware-compatibility.]

[photo](https://1drv.ms/u/s!AhY9wDjh7nlcgZ0_a6Twr-Vzkuk83Q?e=iKfbrq)
![Screenshot_39](https://user-images.githubusercontent.com/94975374/198691825-83cc482c-6e09-4745-b83e-bea587088ae1.png)

**Is there a log.txt file next to FanControl.exe with recent date entries?**
[Link it here]

[log](https://1drv.ms/t/s!AhY9wDjh7nlcgZ1ACzwXr413mDLe-g?e=3EisHH)
[log - copy.txt](https://github.com/Rem0o/FanControl.Releases/files/9890489/log.-.kopie.txt)

[whole V130 folder](https://1drv.ms/u/s!AhY9wDjh7nlcgZpePmuCGLRkxhpl8w?e=earAxf)
[V130.zip](https://github.com/Rem0o/FanControl.Releases/files/9890482/V130.zip)

**Relevant hardware specs and setup**
[Laptops and most proprietary pre-built desktops are not supported.]

[System Information Report.txt](https://github.com/Rem0o/FanControl.Releases/files/9890549/System.Information.Report.txt)

**Comment by Rem0o** [comment]
You got both an Nvidia card and an AMD card in the same system?

Go in sensor settings, disable the NvApiWrapper section and try to only use the GPU section.

**Comment by Tron-Creeper2** [comment]
Yes, I do. Pretty simple reason too.. because my 2nd (older) monitor
doesn't support DP nor HDMI so I had to put in an old NVIDIA gpu that
supports the necessary connectors.

pá 28. 10. 2022 v 20:04 odesílatel Rem0o ***@***.***> napsal:

**Comment by Rem0o** [comment]
It seems like your "old" Nvidia card doesn't respond well to the API FanControl is using. You got tons of API initialization error issues. 

Did you try to "Go in sensor settings, disable the NvApiWrapper section and try to only use the GPU section of LHM." ?

**Comment by Tron-Creeper2** [comment]
[image: Screenshot_40.png]
Well.. now I did. seems to work just fine.. at least for now

so 29. 10. 2022 v 18:19 odesílatel Rem0o ***@***.***> napsal:
